File: app/src/main/java/com/amazon/tv/leanbacklauncher/widget/EditModeViewActionListener.java
package com.amazon.tv.leanbacklauncher.widget;

public interface EditModeViewActionListener {
    void onEditModeExitTriggered();

    void onFocusLeavingEditModeLayer(int i);

    String onPrepForUninstall();

    void onUninstallComplete();

    void onUninstallFailure();
}

File: app/src/main/java/com/amazon/tv/leanbacklauncher/widget/EditModeViewActionListener.kt
package com.amazon.tv.leanbacklauncher.widget

interface EditModeViewActionListener {
    fun onEditModeExitTriggered()
    fun onFocusLeavingEditModeLayer(i: Int)
    fun onPrepForUninstall(): String?
    fun onUninstallComplete()
    fun onUninstallFailure()
}